packages/relay/src/lib/services/hbarLimitService/index.ts

```diff
@@ -203,19 +209,15 @@ export class HbarLimitService implements IHbarLimitService {
     //      In such cases, it should still be true if spendingPlan.amountSpent === spendingLimit.
     const exceedsLimit =
       spendingLimit.lte(spendingPlan.amountSpent) || spendingLimit.lt(spendingPlan.amountSpent + estimatedTxFee);
-
-    this.logger.trace(
-      `${requestDetails.formattedRequestId} User ${
-        exceedsLimit ? 'has' : 'has NOT'
-      } exceeded HBAR rate limit threshold: user=${user}, amountSpent=${
-        spendingPlan.amountSpent
-      }, estimatedTxFee=${estimatedTxFee}, spendingLimit=${spendingLimit}, spandingPlanId=${
-        spendingPlan.id
-      }, subscriptionTier=${
-        spendingPlan.subscriptionTier
-      }, txConstructorName=${txConstructorName}, mode=${mode}, methodName=${methodName}`,
-    );
-
```

Comment:
The updated code appears to be missing some information, likely due to an oversight during the rebase. Please update it to include this information.